third_party/CMakeLists.txt

```diff
@@ -290,44 +290,61 @@ endif()
 add_library(stb INTERFACE)
 target_include_directories(stb INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/stb")
 
+
+# On an x86_64 macbook, AVX2 is selected, but then I get an error during the final link
```

Comment:
Same as above. We either need to tackle this or reword the comment. Esp the "I..." part.

Reply:
fixed